src/test/java/com/google/sps/UserUtilTest.java

```diff
@@ -19,17 +19,18 @@
 import com.google.appengine.api.datastore.Entity;
 import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
 import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import java.util.ArrayList;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.After;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
-import com.google.sps.UserUtils;
+import com.google.sps.utils.UserUtils;
 
 /** tests the IsValidGameName function */
 @RunWith(JUnit4.class)
-public final class GetEntityFromDatastoreTest {
+public final class UserUtilTest {
```

Comment:
Now that we are testing multiple functions in one test class, we should do one of two things:

Make multiple classes within this method, each class being named the method it is testing (whoops, I might have recommended otherwise before, but I'm reversing course now that we're testing multiple methods in a single class lol).

Prepend each method with the name of the function it is testing. 

I prefer the former option but this is my fault for recommending naming the class after the util and putting all method test cases in one test class. I think it makes more sense to split tests by method name into their own test classes and then have one test file per utility class.

Reply:
disregard this message, I'm sending something else in the group chat.